MC/bin/tests/embedding_test.sh

```diff
@@ -15,21 +15,17 @@
 
 # ----------- START ACTUAL JOB  -----------------------------
 
-NSIGEVENTS=${NSIGEVENTS:-20}
-NTIMEFRAMES=${NTIMEFRAMES:-5}
-NWORKERS=${NWORKERS:-8}
-NBKGEVENTS=${NBKGEVENTS:-20}
+NSIGEVENTS=${NSIGEVENTS:-1}
+NTIMEFRAMES=${NTIMEFRAMES:-1}
+NWORKERS=${NWORKERS:-1}
 MODULES="--skipModules ZDC"
-SIMENGINE=${SIMENGINE:-TGeant4}
+SIMENGINE=${SIMENGINE:-TGeant3}
 PYPROCESS=${PYPROCESS:-ccbar} #ccbar, bbar, ...
 SEED=${SEED:+-seed $SEED}
 
 export ALICEO2_CCDB_LOCALCACHE=$PWD/.ccdb
 
 # create workflow
 ${O2DPG_ROOT}/MC/bin/o2dpg_sim_workflow.py -eCM 5020 -col pp -gen pythia8 -proc ${PYPROCESS} \
-                                           -colBkg PbPb -genBkg pythia8 -procBkg "heavy_ion" \
-                                           -tf ${NTIMEFRAMES} -nb ${NBKGEVENTS}              \
-                                           -ns ${NSIGEVENTS} -e ${SIMENGINE}                 \
-                                           -j ${NWORKERS} --embedding -interactionRate 50000 \
-                                           --include-analysis -run 310000 ${SEED}
+                                           -tf ${NTIMEFRAMES} -ns ${NSIGEVENTS} -e ${SIMENGINE} \
+                                           -j ${NWORKERS} --include-analysis -run 310000 ${SEED}
```

Comment:
this is no longer an embedding test? so maybe the filename should be renamed. Also better to give the interaction rate on the command line and specify a fixed seed (at the moment it's random which is not optimal for reproducable testing)

Reply:
We can of course leave it as an embedding test, namely 1 pp into 1 PbPb in that case?